docs/contributing.rst

```diff
@@ -0,0 +1,104 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/ryanhiebert/tox-travis/issues.
+
+If you are reporting a bug, please include:
+
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "feature"
+is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+Tox Travis could always use more documentation, whether as part of the
+official docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at https://github.com/ryanhiebert/tox-travis/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `tox-travis` for local development.
+
+1. Fork the `tox-travis` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/tox-travis.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper
+   installed, this is how you set up your fork for local development::
+
+    $ mkvirtualenv tox-travis
+    $ cd tox-travis/
+    $ python setup.py develop
```

Comment:
Tests are run using tox, so as long as you have tox installed, you shouldn't need this. There's reasons it might be helpful, but understanding them is something probably left to the reader to figure out.

I'd just take this out, with a comment about making sure that you have tox installed (potentially in a virtualenv).